src/Makefile.am

```diff
@@ -1259,6 +1260,17 @@ write_gcm_la_LIBADD += $(BUILD_WITH_LIBCURL_LIBS)
 endif
 endif
 
+pkglib_LTLIBRARIES += write_kosak.la
+write_kosak_la_SOURCES = write_kosak.c
```

Comment:
Yeah, I was going to cut over at some point. I wanted to leave write_gcm alone for now, but maybe we're far along enough that I can replace it.